style/client.css

```diff
@@ -1722,6 +1722,10 @@ a.ilink.yours {
 .pm-buttonbar button:hover {
 	background: #DEE4EA;
 }
+.inner-chat-userlist-maximized {
+	position: absolute;
```

Comment:
Wait, `.inner-chat`? I suppose that _does_ make more sense than any of the things I was thinking.

In hindsight, I think unless you also want to update `right:`, I like your transform implementation more than updating `left`.

Reply:
Yeah definitely open to naming suggestions here, my thinking was this was how the chat behaves when the userlist is maximized so I tried to place this class around the userlist-maximized classes, but can also see how it may belong somewhere else. Will update with a right value, I think just right:0px should work.